bonsai/ethereum-relay/tests/e2e_test.rs

```diff
@@ -24,18 +24,23 @@ mod tests {
         },
         Relayer,
     };
-    use bonsai_proxy_contract::ProxyContract;
+    use bonsai_relay_contract::{BonsaiRelay, BonsaiTestRelay, RiscZeroGroth16Verifier};
     use bonsai_sdk::{
         alpha::{Client as BonsaiClient, SdkErr},
         alpha_async::{get_client_from_parts, put_image},
     };
     use ethers::types::{Bytes, H256 as ethers_H256, U256};
+    use ethers_signers::Signer;
     use risc0_zkvm::{MemoryImage, Program, MEM_SIZE, PAGE_SIZE};
     use risc0_zkvm_methods::{SLICE_IO_ELF, SLICE_IO_ID};
     use tokio::time::{sleep, Duration};
 
     const BONSAI_API_URI: &str = "http://localhost:8081";
 
+    fn dev_mode(bonsai_url: String) -> bool {
+        bonsai_url.contains("localhost") || bonsai_url.contains("127.0.0.1")
+    }
+
```

Comment:
This does not need like a reliable way to tell if we are in `dev_mode`. Can we instead use whatever source of information sets the `bonsai_url` to `localhost` as the decide whether we are in dev mode instead? (This comment applies here, but is actually more relevant in `bonsai/ethereum-relay/src/lib.rs`)